sympy/utilities/runtests.py

```diff
@@ -1000,6 +1018,7 @@ def visit_Assert(self, stmt):
         return fix_missing_locations(new_tree)
 
     def test_file(self, filename, sort=True, timeout=False, slow=False, enhance_asserts=False):
+        reporter = self._reporter
```

Comment:
Is there a reason for this change, other than having a shorthand?
Because usually it's better to stick with the ```self``` prefix: it allows understanding the code without referring back to the assignment. The more the textual length of a scope, the more this applies - here, it's medium-important, and longer ```self._reporter```than ```reporter``` be damned.
(Just my 5 cent, YMMV. Not a reason for rejection.)